src/site/content/en/blog/blibli/index.md

```diff
@@ -0,0 +1,155 @@
+  <figcaption class="w-figcaption">
+    A lightweight page that is three times faster than the old.
+  </figcaption>
+</figure>
+
+On subsequent pages, Blibli prompts users to add the PWA shortcut
+to their home screen.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot" style="width:800px;">
+    <source src="https://storage.googleapis.com/web-dev-assets/blibli/save-to-home-screen.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/blibli/save-to-home-screen.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    The add to home screen prompt.
+  </figcaption>
+</figure>
+
+If users lose the network connection, they are presented with an elegant offline
+fallback page.
+
+<figure class="w-figure">
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/biDb6f1owQWxmRNXUauu.png", alt="An elegent offline fallback page.", width="800", height="468" %}
+  <figcaption class="w-figcaption">
+    An elegent offline fallback page.
+  </figcaption>
+</figure>
+
+Blibli's lite page created a first impression that Blibli.com is fast and
+responsive. Moreover, the lite page streamlined the number of features by only
+focusing on those that deliver the value proposition of Blibli.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot"  style="width:800px;">
+    <source src="https://storage.googleapis.com/web-dev-assets/blibli/responsive-comparison.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/blibli/responsive-comparison.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Comparison of loading times before and after streamlining.
+  </figcaption>
+</figure>
+
+Blibli's PWA is installable, giving it an app-like look and feel. It is also
+fast and reliable which was achieved using service worker caching and graceful
+offline fallback.
+
+<figure class="w-figure">
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/ONCtI6MryOS5wCLPd3Ep.jpeg", alt="The offline fallback", width="800", height="1778" %}
+  <figcaption class="w-figcaption">
+    The offline fallback.
+  </figcaption>
+</figure>
+
+This comes as a delight for web users, a large proportion of whom use mid-tier
+devices. Since Blibli's PWA is less than 1MB in size (that's a whopping 24 times
+lighter than their Android app!), there is much less friction for users to
+install it. It also makes it easier for Blibli to retain users while providing
+them with a seamless app-like experience.
+
+### Set performance budgets
+
+Blibli made performance budgeting as a performance standard within the tech
+development and management, where the standard was reviewed on a regular basis
+and to be set as the KPI of internal tech development, along with the guidance
+of top management to ensure the best platform for all of our users.
```

Comment:
```suggestion
Blibli made performance budgeting a performance standard within the tech
development and management, where the standard was reviewed on a regular basis. 
It's set as the KPI of internal tech development, with the guidance
of top management to ensure the best platform for all of our users.
```

Reply:
I'm not sure if this is better but this is a really big, and somewhat convoluted sentence.